sessions/startup.go

```diff
@@ -0,0 +1,37 @@
+package sessions
+
+import (
+	"context"
+	"net/http"
+	"os"
+)
+
+const startupSessionIDKey = "BUGSNAG_STARTUP_SESSION_ID"
+
+// SendStartupSession is called by Bugsnag on startup, which will send a
+// session to Bugsnag and return a context to represent the session of the main
+// goroutine. This is the session associated with any fatal panics that are
+// caught by panicwrap.
+func SendStartupSession(config *SessionTrackingConfiguration) context.Context {
+	ctx := context.Background()
+	session := newSession()
+	if !config.IsAutoCaptureSessions() || isApplicationProcess(session) {
+		return ctx
+	}
+	publisher := &publisher{
+		config: config,
+		client: &http.Client{Transport: config.Transport},
+	}
+	go publisher.publish([]*Session{session})
+	return context.WithValue(ctx, contextSessionKey, session)
+}
+
+// Checks to see if this is the application process, as opposed to the process
+// that monitors for panics
+func isApplicationProcess(session *Session) bool {
+	// Application process is run first, and this will only have been set when
+	// the monitoring process runs
+	envID := os.Getenv(startupSessionIDKey)
+	os.Setenv(startupSessionIDKey, session.ID.String())
```

Comment:
It might be better not to store the session ID in the environment as we might be tempted to use it. I believe this is only used to detect if we are the monitoring process or not? Could even just look for the same cookie values https://github.com/ConradIrwin/panicwrap/blob/master/panicwrap.go#L25